src/send_packets.c

```diff
@@ -999,7 +998,7 @@ static void calc_sleep_time(tcpreplay_t *ctx, struct timespec *pkt_ts_delta,
         if (timescmp(pkt_ts_delta, time_delta, >)) {
             /* pkt_time_delta has increased, so handle normally */
             timessub(pkt_ts_delta, time_delta, &nap_for);
-            TIMESPEC_SET(&nap_for, &ctx->nap);
+            TIMESPEC_SET(&ctx->nap, &nap_for);
```

Comment:
it's a small bug, left argument will be set to the values of the right argument. The output variable in this case is ctx->nap